packages/backend/src/server/api/endpoints/notes/search.ts

```diff
@@ -1,5 +1,5 @@
 import $ from 'cafy';
-import es from '../../../../db/elasticsearch';
+import searchClient from '../../../../db/searchClient';
```

Comment:
Use `@/` for the root directory of the backend package like this:
```suggestion
import searchClient from '@/db/searchClient';
```